Dockerfile

```diff
@@ -261,9 +261,9 @@ FROM scratch AS centos7
 COPY --from=centos7-build /tmp/google-cloud-ops-agent.tgz /google-cloud-ops-agent-centos-7.tgz
 COPY --from=centos7-build /google-cloud-ops-agent*.rpm /
 
-FROM scratch AS centos8
-COPY --from=centos8-build /tmp/google-cloud-ops-agent.tgz /google-cloud-ops-agent-centos-8.tgz
-COPY --from=centos8-build /google-cloud-ops-agent*.rpm /
+FROM scratch AS rockylinux8
```

Comment:
Our infrastructure actually relies on this name being `centos8`. We would need to fix it on that side first, and then we can do this rename. Renamed back to `centos8` in #377.